runtime/router/router.go

```diff
@@ -153,3 +159,20 @@ func (r *Router) allowed(path, reqMethod string) string {
 
 	return strings.Join(allow, ", ")
 }
+
+func (r *Router) isWhitelistedPath(path string) bool {
+	for _, whitelistedPath := range r.WhitelistedPaths {
+		whitelistedPathTokens := strings.Split(whitelistedPath, "/")
+		pathTokens := strings.Split(path, "/")
+		if len(whitelistedPathTokens) != len(pathTokens) {
+			continue
+		}
+		for i, token := range whitelistedPathTokens {
+			if pathTokens[i] != token && token[0] != ':' {
```

Comment:
no use of continue here.. irrespective you are continuing.

Reply:
what?

Reply:
Fixed